### PR TITLE
SUR-256 Update tabs component as per new mockup

### PR DESCRIPTION
**Pull request by JelenaTakac**
### Description

# Tabs Component Documentation

## Description

The `Tabs` component is a flexible and customizable component built with Tailwind CSS. It supports various orientations, variants, sizes, and additional properties to suit different use cases.

### `orientation`
- **Type:** `string`
- **Default:** `"horizontal"`
- **Description:** Defines the orientation of the tabs. Options include:
  - `"horizontal"`
  - `"vertical"`

### `variant`
- **Type:** `string`
- **Default:** `"pill"`
- **Description:** Defines the style variant of the tabs. Options include:
  - `"pill"`
  - `"rounded"`
  - `"underline"`

### `size`
- **Type:** `string`
- **Default:** `"sm"`
- **Description:** Defines the size of the tabs. Options include:
  - `"xs"`
  - `"sm"`
  - `"md"`
  - `"lg"`

### `iconPosition`
- **Type:** `string`
- **Default:** `"left"`
- **Description:** Defines the position of the icon inside the tab. Options include:
  - `"left"`
  - `"right"`

### `className`
- **Type:** `string`
- **Description:** Additional custom classes to be added to the tabs.

### `width`
- **Type:** `string`
- **Default:** `"auto"`
- **Description:** Defines the width of the tabs. Options include:
  - `"auto"`
  - `"full"`

### `activeItem`
- **Type:** `string|null`
- **Default:** `null`
- **Description:** The currently active item in the group.

### `onChange`
- **Type:** `Function`
- **Description:** Callback when the active item changes.

### `children`
- **Type:** `ReactNode`
- **Description:** The content to be displayed inside the tabs.

### `slug`
- **Type:** `string`
- **Description:** Unique identifier for the tab.

### `text`
- **Type:** `string`
- **Description:** Text to be displayed inside the tab.

### `icon`
- **Type:** `ReactNode`
- **Description:** An optional icon to be displayed inside the tab.

### `className`
- **Type:** `string`
- **Description:** Additional custom classes to be added to the tab.

### `disabled`
- **Type:** `boolean`
- **Default:** `false`
- **Description:** Defines whether the tab is disabled.

### `badge`
- **Type:** `ReactNode`
- **Description:** An optional badge to be displayed alongside the tab text.

### Basic Example

```jsx
import Tabs from './Tabs';
import { ReactComponent as ExampleIcon } from './icons/example.svg';

const App = () => (
  <div>
    <Tabs.Group variant="rounded" size="md" orientation="horizontal">
      <Tabs.Tab slug="tab1" text="Tab 1" icon={<ExampleIcon />} />
      <Tabs.Tab slug="tab2" text="Tab 2" icon={<ExampleIcon />} />
    </Tabs.Group>
  </div>
);

export default App;
```

### Checklist:

-   [ ] My code is tested
-   [ ] My code passes the PHPCS tests
-   [ ] I've created the npm build.
-   [ ] My code follows accessibility standards 
-   [ ] My code has proper inline documentation 
-   [ ] I've included any necessary tests 
-   [ ] I've included developer documentation 
-   [ ] I've added proper labels to this pull request

### CODE REVIEW

**Review by jaieds — CHANGES_REQUESTED** [review]
UI not matching with the Figma mockup. 

👉 https://d.pr/i/HolCWo

File: src/components/tabs/tabs.jsx [hunk 1]
```diff
@@ -123,9 +140,10 @@ const Tab = ( props, ref ) => {
 
 	// Determine size classes.
 	const sizes = {
-		sm: 'p-1 text-sm [&>svg]:h-4 [&>svg]:w-4',
-		md: 'p-2 text-base [&>svg]:h-5 [&>svg]:w-5',
-		lg: 'p-2.5 text-lg [&>svg]:h-6 [&>svg]:w-6',
+		xs: 'px-1.5 py-0.5 text-xs [&>svg]:h-3 [&>svg]:w-3',
+		sm: variant === 'underline' ? 'py-1.5 text-sm [&>svg]:h-4 [&>svg]:w-4' : 'px-3 py-1.5 text-sm [&>svg]:h-4 [&>svg]:w-4',
```
Comment (ravindra114) [inline src/components/tabs/tabs.jsx:144]: You can use size-4 instead of [&>svg]:h-4 [&>svg]:w-4

**Review by vrundakansara — APPROVED** [review]

File: src/components/tabs/tabs.jsx [hunk 2]
```diff
@@ -190,6 +198,11 @@ const Tab = ( props, ref ) => {
 			onClick={ handleClick }
 			{ ...rest }
 		>
+			{
+				activeItem === slug && variant === 'underline'
+					? <span className="absolute right-0 left-0 -bottom-[1px] h-[1px] bg-border-interactive"></span>
```
Comment (jaieds) [inline src/components/tabs/tabs.jsx:203]:
```suggestion
					? <span className="absolute right-0 left-0 -bottom-px h-px bg-border-interactive"></span>
```

File: src/components/tabs/tabs.jsx [hunk 3]
```diff
@@ -175,7 +183,7 @@ const Tab = ( props, ref ) => {
 		className
 	);
 
-	const iconParentClasses = twMerge( 'flex items-center gap-1' );
+	const iconParentClasses = cn( 'flex items-center gap-1' );
```
Comment (jaieds) [inline src/components/tabs/tabs.jsx:186]:
```suggestion
	const iconParentClasses = 'flex items-center gap-1';
```

File: src/components/tabs/tabs.jsx [hunk 4]
```diff
@@ -134,22 +151,17 @@ const Tab = ( props, ref ) => {
         orientation === 'vertical' ? 'w-full justify-between' : '';
 
 	// Base classes for the Tab.
-	const baseClasses = `bg-transparent text-text-tertiary cursor-pointer flex items-center justify-center transition-colors duration-200 ${ fullWidth } ${ orientationClasses }`;
+	const baseClasses = `relative border-none bg-transparent text-text-tertiary cursor-pointer flex items-center justify-center transition-colors duration-200 ${ fullWidth } ${ orientationClasses }`;
```
Comment (jaieds) [inline src/components/tabs/tabs.jsx:154]: Try not to use Template string for class names, use the `cn` function instead.

**Review by ravindra114 — COMMENTED** [review]
I think on hover it should show background to tab?
image

not able to see this

https://github.com/user-attachments/assets/357fc092-688b-4857-8ab7-b13500b1c280

**Review by jaieds — CHANGES_REQUESTED** [review]
@JelenaTakac, Please check the screenshot 👉 https://d.pr/i/Rtcakc

**Review by jaieds — APPROVED** [review]

**Pull request merged**